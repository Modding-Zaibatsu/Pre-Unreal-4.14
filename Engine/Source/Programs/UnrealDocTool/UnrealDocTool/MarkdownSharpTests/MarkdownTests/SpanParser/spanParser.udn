Normal normal _italic [Simple link with underscore_](http://www.google.com).

Normal text [Link with **bold text**](http://epicgames.com/).

Italicized link start _[_ and bolded link end __]__.

Bolded wrong __reference [link]__.

Italicized image start _![_ and bolded link end __]__.

__Te_st__ image in li_nk [![ue_4.png](ue_4.png)(w:600)](ue_4.png).

Ig_no_re midword tags.

\_This_is_not_em_text\_

_This_is_em_text_