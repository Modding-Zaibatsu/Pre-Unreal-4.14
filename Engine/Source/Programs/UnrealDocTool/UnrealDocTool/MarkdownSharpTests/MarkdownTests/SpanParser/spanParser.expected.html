<?xml version="1.0" encoding="UTF-8" ?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <p>Normal normal _italic <a href="http://www.google.com" >Simple link with underscore_</a>.</p>
        
        <p>Normal text <a href="http://epicgames.com/" >Link with <strong>bold text</strong></a>.</p>
        
        <p>Italicized link start <em>[</em> and bolded link end <strong>]</strong>.</p>
        
        <p>Bolded wrong <strong>reference [link]</strong>.</p>
        
        <p>Italicized image start <em>![</em> and bolded link end <strong>]</strong>.</p>
        
        <p><strong>Te_st</strong> image in li_nk <a href="./../../images\Temp\ue_4.png" >
            <img alt="ue_4.png" width="600" src="./../../images\Temp\ue_4.Jpeg" /></a>.</p>
        
        <p>Ig_no_re midword tags.</p>
        
        <p>_This_is_not_em_text_</p>

        <p><em>This_is_em_text</em></p>
    </body>
</html>
